### PR TITLE
Use the scheduler everywhere (but passthru outside the adaptive test)

### PR DESCRIPTION
**Pull request by mxdvl**
## What does this change?

We use the scheduler to boot our scripts for all users.

We refactored the logic so that imports start before polyfills are available, but execution awaits the polyfill.io callback.

## Why?

This allows use to compare only different settings applied to the scheduler, rather than two distinct `index.ts` files with distince boot strategies for the upcoming adaptive test variant.

### CODE REVIEW

**Comment by mxdvl** [comment]
We’ve got confirmation from commercial analysts that this did not have any adverse effects on revenue.

File: dotcom-rendering/src/server/htmlPageTemplate.ts [hunk 1]
```diff
@@ -256,7 +256,7 @@ https://workforus.theguardian.com/careers/product-engineering/
 
                 <script>
                     window.guardian = ${windowGuardian};
-                    window.guardian.queue = []; // Queue for functions to be fired by polyfill.io callback
+                    window.guardian.onPolyfilledCallbacks = []; // Queue for functions to be fired by polyfill.io callback
```
Comment (sndrs) [inline dotcom-rendering/src/server/htmlPageTemplate.ts:259]: wasn't this going to break a number of things?

Comment (sndrs) [inline dotcom-rendering/src/server/htmlPageTemplate.ts:259]:
```suggestion
                    window.guardian.queue = []; // Queue for functions to be fired by polyfill.io callback
```

File: dotcom-rendering/src/server/htmlPageTemplate.ts [hunk 2]
```diff
@@ -289,7 +289,7 @@ https://workforus.theguardian.com/careers/product-engineering/
                     function guardianPolyfilled() {
                         window.guardian.polyfilled = true;
                         if (window.guardian.mustardCut === false) {
-                            window.guardian.queue.forEach(function(startup) { startup() })
+                            window.guardian.onPolyfilledCallbacks.forEach(function(callback) { callback() })
```
Comment (sndrs) [inline dotcom-rendering/src/server/htmlPageTemplate.ts:292]:
```suggestion
                            window.guardian.queue.forEach(function(callback) { callback() })
```

File: dotcom-rendering/src/server/htmlPageTemplate.ts (hunk 1 above)
Comment (mxdvl) [reply dotcom-rendering/src/server/htmlPageTemplate.ts:259]: How did this not get caught? 🤔

**Review by AshCorr — APPROVED** [review]

**Pull request merged**